exonum/src/api/websocket.rs

```diff
@@ -87,15 +86,18 @@ impl TransactionFilter {
 /// Summary about a particular transaction in the blockchain (without transaction content).
 #[derive(Debug, Serialize, Deserialize, PartialEq)]
 pub struct CommittedTransactionSummary {
-    tx_hash: Hash,
+    /// Transaction identifier.
+    pub tx_hash: Hash,
     /// ID of service.
     pub service_id: u16,
     /// ID of transaction in service.
     pub message_id: u16,
-    #[serde(with = "TxStatus")]
-    status: TransactionResult,
-    location: TxLocation,
-    proof: ListProof<Hash>,
+    /// Result of transaction execution.
+    pub status: ExecutionOutcome,
+    /// Transaction location in the blockchain.
+    pub location: TxLocation,
+    /// Proof of existence.
+    pub proof: ListProof<Hash>,
```

Comment:
FYI I've renamed it to `location_proof` in #1386 for consistency with REST API response.